contrib/bundle/install

```diff
@@ -1,7 +1,7 @@
 set -x
 install $SELINUX -d -m 755 "$DESTDIR$CNIDIR"
 install $SELINUX -D -m 755 -t "$DESTDIR$OPT_CNI_BIN_DIR" cni-plugins/*
-install $SELINUX -D -m 644 -t "$DESTDIR$CNIDIR" contrib/10-crio-bridge.conf
+install $SELINUX -D -m 644 -t "$DESTDIR$CNIDIR" contrib/10-crio-bridge.conflist
```

Comment:
weirdly enough the path here was wrong. it should be contrib/cni/10-crio-bridge.conflist . I'm not sure why this worked before

Reply:
it's causing [this](https://github.com/cri-o/cri-o/actions/runs/3751631905/jobs/6373420977) error

Reply:
Ya even I saw this error in the build now. Not sue how it was working up until now 😢

Reply:
Is this issue planned to be fixed? 🙏 

Hitting this in the cri-tools CI
https://github.com/kubernetes-sigs/cri-tools/pull/1053#issuecomment-1368555519

Reply:
I want to take this one.

@haircommander can you confirm whether you have started working on this or not?

Reply:
Made a temp fix on [this](https://github.com/cri-o/cri-o/pull/6476) PR for this issue.

cc @haircommander @AkihiroSuda

Reply:
thanks for taking it on @amarlearning !